tests/test_misc.py

```diff
@@ -0,0 +1,22 @@
+import ast
+import unittest
+import sys
+
+import astor
+
+
+class GetSymbolTestCase(unittest.TestCase):
+
+    def test_get_mat_mult(self):
+        if sys.version_info < (2, 7):
+            # We can't use `@unittest.skipIf` or `raise skipTest` in
```

Comment:
We are using unittest2 in Python 2.6. If you import unittest2 conditionally, you can use `unittest.skipIf` in Python 2.6:

```py
try:
    import unittest2 as unittest
except ImportError:
    import unittest
```